data/coalition/coalition missions.txt

```diff
@@ -1483,8 +1483,8 @@ mission "Coalition: Submarines 2"
 				`	"Sure, why not? Probably beats waiting around and checking shops over and over."`
 				`	"Thanks for the invitation, but I'm not really interested."`
 					goto refuse
-			`	After all the submarines are out of your cargo hold, they're placed atop large truck-like vehicles, but with the wheels in a system of continuous tracks. You and Apri'ie board one of the trucks, and begin a trip down to the shore below. The truck follows the water out of the fjord until it reaches a small commercial port with a large set of docks. Once there, Apri'ie introduces you to some of his colleagues who are already waiting to begin the tests. The subs are activated before they are off the trucks, and some of the Kimek begin to test basic functions, each working on a different computer to control their respective submarine.`
-			`	Once they're all in the water, everyone boards a large barge, which follows the submarines several kilometers off the coast until you can no longer see the shore. At this point they stop, and one of the submarines submerges. For about ten minutes, nothing happens; the other four submarines just wait around while the Kimek conversing among themselves as they analyze readings of the submerged craft. Eventually, it touches the bottom and begins the trip back up. It emerges in roughly twenty minutes, looking just like it did previously, save for a long strip of kelp stuck to one of its sides. The Kimek make sure the sub suffered no damage, and proceed to test the other ones. After each one has made the trip to the bottom and back several times, the submarines and the barge head back to the docks.`
+			`	After all the submarines are out of your cargo hold, they're placed atop large truck-like vehicles, but with the wheels in a system of continuous tracks. You and Apri'ie board one of the trucks, and begin a trip down to the shore below. The truck follows the water out of the fjord until it reaches a small commercial port with a large set of docks. Once there, Apri'ie introduces you to some of his colleagues who are already waiting to begin the tests. The subs are activated before they are off the trucks, and some of the Kimek begin to test basic functions.`
```

Comment:
```suggestion
			`	After all the submarines are out of your cargo hold, they're placed atop trucks with heavy tracks instead of wheels. You and Apri'ie board one of the trucks and begin a trip down to the shore below, reaching a small commercial port with a large set of docks. Once there, Apri'ie introduces you to some of his colleagues who are already waiting to begin the tests. The subs are activated before they are off the trucks, and some of the Kimek begin to test basic functions.`
```